pages/docs/integrations/grouparoo-intercom.mdx

```diff
@@ -11,132 +11,218 @@ Grouparoo's [Intercom](https://www.intercom.com/) Plugin enables you to export c
 
 This guide will show you how to work with the Intercom Plugin to [create a Destination](#create-an-intercom-destination) to export your data.
 
-<PluginDocsInstallation plugin="Intercom" />
+## Overview
 
-<PluginDocsCreateApp plugin="Intercom" />
+Intercom is a messaging platform for customer communications. You can use Intercom to stay in touch with customers. Grouparoo can help you to sync data from your warehouse to Intercom easily.
 
-<ValidateAndApplyConfig headingLevel="h3" />
+### Destinations
 
-## Create an Intercom Destination
+<table className="docTable">
+  <tbody>
+    <tr>
+      <td>Intercom Export Contacts</td>
+      <td>Export Records to contacts in an Intercom account.</td>
+    </tr>
+  </tbody>
+</table>
```

Comment:
I feel like this table needs a header to explain what's being shown:

| Grouparoo Destination | What it Does |
| --- | --- |
| Intercom Export Contacts | Export Records to contacts in an Intercom account. |

Reply:
Agreed!

Reply:
I added a header row for this table and fixed a text body header further down that wasn't rendering markdown properly.